patch-package
--- a/node_modules/react-native-google-places-autocomplete/GooglePlacesAutocomplete.js
+++ b/node_modules/react-native-google-places-autocomplete/GooglePlacesAutocomplete.js
@@ -16,6 +16,7 @@ import {
 } from 'react-native';
 import Qs from 'qs';
 import debounce from 'lodash.debounce';
+import ImageResizeMode from 'react-native/Libraries/Image/ImageResizeMode';
 
 const WINDOW = Dimensions.get('window');
 
@@ -609,7 +610,7 @@ export default class GooglePlacesAutocomplete extends Component {
       >
         <Image
           style={[defaultStyles.powered, this.props.styles.powered]}
-          resizeMode={Image.resizeMode.contain}
+          resizeMode={ImageResizeMode.contain}
           source={require('./images/powered_by_google_on_white.png')}
         />
       </View>
